src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java

```diff
@@ -150,6 +150,11 @@ public static MultipleClassification of(String strategy) {
     private Integer numKFold = -1;
 
     /**
+     * Random seed is used to split training set and test set, default value is 1500890350L.
+     */
+    private Long baggingSampleSeed = Long.valueOf(1500890350L);
```

Comment:
Yes, it's better to have an fallback measure. Changed to  baggingSeed-revise branch.